src/server/scripts/EasternKingdoms/SunwellPlateau/boss_kiljaeden.cpp

```diff
@@ -177,7 +177,7 @@ struct npc_kiljaeden_controller : public NullCreatureAI
     void ResetOrbs()
     {
         for (uint8 i = 0; i < 4; ++i)
-            if (GameObject* orb = ObjectAccessor::GetGameObject(*me, instance->GetGuidData(DATA_ORB_OF_THE_BLUE_DRAGONFLIGHT_1 + i)))
+            if (GameObject* orb = instance->GetGameObject(i))
```

Comment:
Is this correct?
I would have expected that the for loop starts at the define value and not at 0

Reply:
I'm not sure if I understand your question

![image](https://github.com/user-attachments/assets/b516cad2-5be7-4ee1-b17c-46dc0af02886)

Reply:
Alright, It just feels a bit like magic number here. So instead of i=0 could use the define as limits maybe?

Reply:
Or like it was before, "DATA_.. +i)"